### PR TITLE
Chore/update UI forsales

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: src/commands/renderSalesMessage.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
 import { Message, MessageAttachment } from "discord.js"
-import { sales } from "types/sales"
+import { NftSales } from "types/sales"
```
Comment (trkhoi) [inline src/commands/renderSalesMessage.ts:2]: Already create function in `/utils/sales.ts` then fix the logic in `/utils/sales.ts`
If you want to use, import from "utils/sales" => can check if the function in `utils/sales.ts` is correct or not

**Review by trkhoi — CHANGES_REQUESTED** [review]

File: src/commands/renderSalesMessage.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 import { composeEmbedMessage } from "utils/discordEmbed"
 import * as Canvas from "canvas"
```
Comment (trkhoi) [inline src/commands/renderSalesMessage.ts:4]: this command is not used so fix logic here does not help

**Review by trkhoi — CHANGES_REQUESTED** [review]
